spec/receptacle_spec.rb

```diff
@@ -202,6 +214,25 @@ def delete(id:)
           end
         end
       end
+
+      describe ".update" do
+        let(:add10_flip_unscoped_and_multiply_result_wrapper) do
```

Comment:
how about naming this add10_wrapper? it does nothing with uncoped or multiplying